### PR TITLE
Webroot.WebrootSecureAnywhere version 9.0.31.86

### PR DESCRIPTION
**Pull request by GFOXNEXON**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56470)

### CODE REVIEW

File: manifests/w/Webroot/WebrootSecureAnywhere/9.0.31.86/Webroot.WebrootSecureAnywhere.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Webroot.WebrootSecureAnywhere
+PackageVersion: 9.0.31.86
+InstallerSwitches:
+  Silent: /silent
```
Comment (GFOXNEXON) [inline manifests/w/Webroot/WebrootSecureAnywhere/9.0.31.86/Webroot.WebrootSecureAnywhere.installer.yaml:7]: SilentWithProgress: /silent
  Custom: /exeshowaddremove /lang=en

**Review by GFOXNEXON — COMMENTED** [review]
X64 bit version of Webroot

**Comment by GFOXNEXON** [comment]
SilentWithProgress: /silent
Custom: /exeshowaddremove /lang=en

64 bit version to silently install and have REG or GPO apply product key after install

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
"manifests/w/Webroot/WebrootSecureAnywhere/readme" needs to be removed.

**Pull request closed without merging**